libs/designer-ui/src/lib/panel/panelpivot.tsx

```diff
@@ -56,17 +56,33 @@ export const PanelPivot = ({ isCollapsed, tabs, selectedTab, onTabChange, nodeId
       >
         {Object.entries(tabs).map(([name, { visible, tabErrors, title }]) => {
           return visible && tabErrors?.[nodeId] ? (
-            <PivotItem key={name} itemKey={name} headerText={title} onRenderItemLink={customRenderer} />
+            <PivotItem key={name} itemKey={name} headerText={title} onRenderItemLink={customRendererWithErrors} />
```

Comment:
@rllyy97 to check if nothing breaks here

Reply:
Just ran it locally, it's not breaking but there is a minor css interruption with adding this extra div for the automation-id in between the custom render content and the parent, but honestly it's very similar without the styling (just a small position change) and not changing any functionality, so I'm fine with it

### CURRENT
![image](https://github.com/Azure/LogicAppsUX/assets/25409734/9c028a17-0330-48cf-b3a8-411dce864663)

### THIS PR
![image](https://github.com/Azure/LogicAppsUX/assets/25409734/23ffa5e2-5ca7-42cf-92af-936fefa63e8e)

Reply:
Ok, sounds good. Thanks @rllyy97 

@preetriti1 Can you re-review/approve as appropriate?